crates/geometry/src/convex_hull.rs

```diff
@@ -0,0 +1,53 @@
+use linear_algebra::Point2;
+use nalgebra::Matrix2;
+
+pub fn reduce_to_convex_hull<Frame>(
+    points: &[Point2<Frame>],
+    only_top_half: bool,
```

Comment:
Please use a custom enum here instead to be more descriptive in the function call. E.g. `Range::Full` / `Range::OnlyTopHalf`

Reply:
Done